lib/clover_field.cpp

```diff
@@ -373,6 +373,23 @@ namespace quda {
     checkCudaError();
   }
 
+  void cudaCloverField::prefetch(QudaFieldLocation mem_space, cudaStream_t stream) const
```

Comment:
For clover I'm thinking we should add a specialization of `prefetch` that allows us to specify whether to fetch the clover field or its inverse as opposed to only acting on both.